scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -45,6 +82,23 @@ class ReducerEstimator {
 
 }
```

Comment:
I think `ReducerEstimator` should be a trait, not a class.

I just caught a bug (in this version of the code) in which every single reducer estimator's `estimateReducers` method returns `None` because I forgot to override the method!

Such a change would prevent this kind of bug.